meta-boxes.php

```diff
@@ -1,20 +1,139 @@
 <?php
+/**
+ * File containing meta box callback functions
+ */
 
-function nprstory_publish_meta_box($post) {
-	$helper_text = 'Push this story to NPR:';
-	$is_disabled = ($post->post_status != 'publish');
-	if ($is_disabled) {
-		$helper_text = 'Publish this story in order to push it to NPR.';
+/**
+ * Output the NPR Story API publishing options metabox for the edit page admin interface
+ *
+ * @param WP_Post $post the WordPress post object.
+ * @see nprstory_save_send_to_api
+ * @see nprstory_save_send_to_org
+ * @see nprstory_save_send_to_one
+ * @see nprstory_save_nprone_featured
+ *
+ * @todo When there is better browser support for input type="datetime-local", replace the jQuery UI and weird forms with the html5 solution. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime-local
+ */
+function nprstory_publish_meta_box( $post ) {
+	$helper_text = __( 'Push this story to NPR:', 'nprapi' );
+	$is_disabled = ( 'publish' !== $post->post_status );
+	$attrs = array( 'id' => 'ds-npr-update-push' );
+
+	if ( $is_disabled ) {
+		$helper_text = __( 'Publish this story in order to push it to NPR.', 'nprapi' );
 	}
-	$attrs = array('id' => 'ds-npr-update-push');
-	if ($is_disabled)
+
+	if (  $is_disabled ) {
 		$attrs['disabled'] = 'disabled';
-?>
+	}
+
+	wp_enqueue_style( 'jquery-ui' );
+	wp_enqueue_style( 'nprstory_publish_meta_box_stylesheet' );
+	wp_enqueue_script( 'nprstory_publish_meta_box_script' );
+
+	?>
 	<div id="ds-npr-publish-actions">
-		<p class="helper-text"><?php echo $helper_text; ?></p>
-<?php
-	submit_button( 'Push to NPR', 'large', 'ds_npr_update_push', false, $attrs );
-?>
+		<ul>
+		<?php
+			// send to the npr api
+			// The meta name here is '_send_to_nprone' for backwards compatibility with plugin versions 1.6 and prior
+			$nprapi = get_post_meta( $post->ID, '_send_to_nprone', true ); // 0 or 1
+			if ( '0' !== $nprapi && '1' !== $nprapi ) { $nprapi = 1; } // defaults to checked; unset on new posts
+
+			// this list item contains all other list items, because their enabled/disabled depends on this checkbox
+			echo '<li>';
+			printf(
+				'<label><input value="1" type="checkbox" name="send_to_api" id="send_to_api" %2$s/> %1$s</label>',
+				__( 'Send to NPR API', 'nprapi' ),
+				checked( $nprapi, '1', false )
+				// @see nprstory_save_send_to_api for a historical note on this metadata name
+			);
+
+			echo '<ul>';
+
+			// send to npr dot org
+			printf(
+				'<li><label><input value="1" type="checkbox" name="send_to_org" id="send_to_org" %2$s/> %1$s</label></li>',
+				__( 'Include for reading on NPR.org', 'nprapi' ),
+				checked( get_post_meta( $post->ID, '_send_to_org', true ), '1', false )
+			);
```

Comment:
Remove this section for now - we are not ready for it.

Reply:
Removed in 493ef78.